src/module.c

```diff
@@ -846,7 +846,7 @@ int64_t commandKeySpecsFlagsFromString(const char *s) {
         else if (!strcasecmp(t,"INSERT")) flags |= CMD_KEY_INSERT;
         else if (!strcasecmp(t,"UPDATE")) flags |= CMD_KEY_UPDATE;
         else if (!strcasecmp(t,"DELETE")) flags |= CMD_KEY_DELETE;
-        else if (!strcasecmp(t,"channel")) flags |= CMD_KEY_CHANNEL;
+        else if (!strcasecmp(t,"shard_channel")) flags |= CMD_KEY_CHANNEL;
```

Comment:
I also didn't understand this change. Channels are not the same as shard channels.

Reply:
my thought was that in the context of key-spec, saying these are pubsub channels is enough, and it is enough to say that they're "channels". i.e. normal PUBLISH channels don't have key-spec, the only reason these have, is because they're sharded.

the main reason i did that was because i wanted all key-spec flags to be short, and one word. i guess that's not a real important concern.
let me know if you feel strongly that this is a mistake, in which case we'll revert it ASAP.

p.s. i also see i used upper-case letters for `RW`, `RO`, and also for `INSERT`, `UPDATE`, but not for channel and incomplete.
unlike COMMAND INFO, in which case they're all lower-case, except `RW`, `RO` which are initials for two words.
i suppose we should consolidate that (COMMAND INFO and module API should be the same), which way do you prefer?

Reply:
i see that `addReplyFlagsForKeyArgs` exposes `CMD_KEY_CHANNEL` and not `CMD_KEY_SHARD_CHANNEL`, and `commandKeySpecsFlagsFromString` is the other way around.
i'm still not certain about their different roles